api/cmd/hub/main.go

```diff
@@ -97,7 +104,7 @@ func main() {
 			} else if u.Port() == "" {
 				u.Host += ":80"
 			}
-			handleHTTPServer(ctx, u, categoryEndpoints, &wg, errc, logger, *dbgF)
+			handleHTTPServer(ctx, u, categoryEndpoints, resourceEndpoints, &wg, errc, logger, *dbgF)
```

Comment:
can we keep all services endpoint in one struct and pass it to the handleHTTPServer??

Reply:
yeah can be passed as a struct